### DISCUSSION TITLE
Send message document with Mimetype and filename

### DISCUSSION THREAD

**Discussion by chaad98** [WhiskeySockets/Baileys#512] [General Questions]
Hi all, how can i send message with document type .pdf / ,xlsx / .pptx and etc. I tried to follow the documentation given and it works but this operation sending document can only be send as in .pdf file even i send it as .xlsx. In other words, i tried to send the .xlsx file and i received from my whatsapp as in .pdf file. I also have mimetype in my source code to declare what document files that im going to send to. Still, its not working. so, that is error to me.

Also the filename is is not working even i edit the filename and it still show as "file". 

How can i solve this sending document other than .pdf and the filename?

**Comment by hamisirizwan** [comment]
also having same issue, Did you find a way through?
